server/ldap_check.rb

```diff
@@ -14,14 +14,19 @@ class LdapCheck
 
   ## Setup the ldap connection object and save the members of the group.
 
-
   def initialize(args={})
 
     # This file will be provided in the build with values
     # suitable for the MCommunity groups.
     @default_conf_file = "./server/local/ldap.yml"
+    @default_cache_seconds = 300
```

Comment:
It's just a heuristic.  5 minutes is long enough to catch someone with a heavy click finger but short enough that the problem will fix itself if you go get a cup of coffee.